### PR TITLE
Finalizing JSON support

### PR DESCRIPTION
**Pull request by davidrabinowitz**

### CODE REVIEW

**Review by suryasoma — APPROVED** [review]

File: spark-bigquery-dsv2/spark-3-bigquery-common/pom.xml [hunk 1]
```diff
@@ -29,12 +29,5 @@
       <version>${spark.version}</version>
       <scope>test</scope>
     </dependency>
-    <dependency>
```
[the remark this answers is not in the document]
Comment (davidrabinowitz) [reply spark-bigquery-dsv2/spark-3-bigquery-common/pom.xml:32]: Due to compilation issues

File: README-template.md [hunk 2]
```diff
@@ -887,6 +887,19 @@ We are considering adding an optional TimeZone property to allow automatically
 When casting to Timestamp TIME have the same TimeZone issues as DATETIME
    </td>
   </tr>
+  <tr valign="top">
+   <td><strong><code>JSON</code></strong>
+   </td>
+   <td><strong><code>StringType</code></strong>
+   </td>
+   <td>Spark has no JSON type. The values are read as String. In order to write JSON back to BigQuery, the following conditions are <b>REQUIRED</b>:
```
[the remark this answers is not in the document]
Comment (davidrabinowitz) [reply README-template.md:895]: It's an implementation detail

**Pull request merged**